spec/reek/smells/attribute_spec.rb

```diff
@@ -47,26 +35,10 @@ class Klass
       ').to_not reek_of(:Attribute)
     end
 
-    it 'records attr attribute in a module' do
+    it 'records attr_writer attribute in a module' do
       expect('
         module Mod
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records attr attribute' do
-      expect('
-        class Klass
-          attr :my_attr
-        end
-      ').to reek_of(:Attribute, name: 'my_attr')
-    end
-
-    it 'records reader attribute' do
-      expect('
-        class Klass
-          attr_reader :my_attr
+          attr_writer :my_attr
         end
```

Comment:
just a little update, i'm digging into this..

the `true` is not reported as `arg_name`
```
[1] pry(#<Reek::Smells::Attribute>)> call_node
=> (send nil :attr
  (sym :my_attr)
  (true))
[2] pry(#<Reek::Smells::Attribute>)> call_node.arg_names
=> [:my_attr, nil]
```
i could hack something like `call_node[2] == "attr" && call_node[4] == :something`

but on the other hand, i have no idea how to check for `(true)`

```
[6] pry(#<Reek::Smells::Attribute>)> call_node[4]
=> (true)
[7] pry(#<Reek::Smells::Attribute>)> call_node[4].class
=> #<Class:0x007fc7f2b12df0>
```